lidiabrowser/lidia/models.py

```diff
@@ -0,0 +1,89 @@
+from django.db import models
+
+
+class Publication(models.Model):
+    zotero_id = models.CharField(max_length=100, unique=True)
+    attachment_id = models.CharField(max_length=16, unique=True, null=True)
+    title = models.CharField(max_length=255, null=True)
+
+    def __str__(self):
+        return self.title or self.zotero_id
+
+
+class Language(models.Model):
+    code = models.CharField(max_length=3, unique=True)
+    name = models.CharField(max_length=100, null=True)
+
+    def __str__(self):
+        return self.name or self.code
+
+
+class Annotation(models.Model):
+    RELATION_TYPE_CHOICES = [
+        ("", "None"),
+        ("contradicts", "Contradicts"),
+        ("generalizes", "Generalizes"),
+        ("invalidates", "Invalidates"),
+        ("specialcase", "Is a special case of"),
+        ("supports", "Supports"),
+    ]
+
+    zotero_id = models.CharField(max_length=100, unique=True, null=False)
+    parent_attachment = models.ForeignKey(Publication, on_delete=models.CASCADE, to_field='attachment_id', blank=True, null=True)
+    textselection = models.TextField(default='') # Limit how much text we store, or limit displayed text?
```

Comment:
Limit because of copyright reasons? I think we should not worry about that now - if this becomes an issue I would suggest to keep it in the database (and allow editors to see it all here) and limit it for external viewers. We could make an "open access" option for publications where a limitation is not needed.